frontend/src/components/CheckInPanel.vue

```diff
@@ -3,26 +3,33 @@
 		<h2 class="text-lg font-bold text-gray-900">
 			Hey, {{ employee?.data?.first_name }} 👋
 		</h2>
-		<div class="font-medium text-sm text-gray-500 mt-1.5" v-if="lastLog">
-			Last {{ lastLogType }} was at {{ lastLogTime }}
-		</div>
-		<Button
-			class="mt-4 mb-1 drop-shadow-sm py-5 text-base"
-			id="open-checkin-modal"
-			@click="checkinTimestamp = dayjs().format('YYYY-MM-DD HH:mm:ss')"
-		>
-			<template #prefix>
-				<FeatherIcon
-					:name="
-						nextAction.action === 'IN'
-							? 'arrow-right-circle'
-							: 'arrow-left-circle'
-					"
-					class="w-4"
-				/>
-			</template>
-			{{ nextAction.label }}
-		</Button>
+		<template v-if="DisableCheckinForMobileSetting.data">
+			<div class="font-medium text-sm text-gray-500 mt-1.5">
+				Checkin is disabled.
```

Comment:
Change this to something more descriptive? Like "Employee check-in has been disabled for the mobile app"